orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/tencentcloud/TencentCloudServerGroupCreator.groovy

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Please do not submit new Groovy code in `src/main`.

Reply:
Changed it to Java code.